docs/csharp/tour-of-csharp/features.md

```diff
@@ -0,0 +1,118 @@
+---
+title: A Tour of C# - Major language areas
+description: New to C#? Learn the basics of the language.
+ms.date: 08/06/2020
+---
+# Major language areas
+
+## Arrays, collections, and LINQ
+
+C# and .NET provide many different collection types. `Arrays` have syntax defined by the language. General collection types are listed in the <xref:System.Collections.Generic?displayProperty=fullName> namespace. Specialized collections include <xref:System.Span%601?displayProperty=nameWithType> for accessing continuous memory on the stack frame, and <xref:System.Memory%601?displayProperty=nameWithType> for accessing continuous memory on the managed heap. All collections, including arrays, <xref:System.Span%601>, and <xref:System.Memory%601> share a unifying principle for iteration. You use the <xref:System.Collections.Generic.IEnumerable%601?displayProperty=nameWithType> interface. This unifying principle means that any of the collection types can be used with LINQ queries or other algorithms. You write methods using <xref:System.Collections.Generic.IEnumerable%601> and those algorithms work with any collection.
```

Comment:
I don't think "Arrays" in the second sentence should be code fenced: there is no such type and it won't be localized, while it seems it should be.